lib/mix/lib/mix/cli.ex

```diff
@@ -73,6 +74,17 @@ defmodule Mix.CLI do
   defp ensure_hex(_task),
     do: Mix.Hex.ensure_updated?()
 
+  defp ensure_no_slashes(task) do
```

Comment:
We can move this check to inside `run_task`, so we can re-use its exception handling.